scripts/gulp/lint.js

```diff
@@ -22,53 +20,63 @@ function _genericLintJs( src ) {
 
 /**
  * Lints the gulpfile for errors.
+ * @returns {Object} An output stream from gulp.
  */
-gulp.task( 'lint:build', () => {
-  _genericLintJs( [
+function lintBuild() {
+  return _genericLintJs( [
     'gulpfile.js',
-    'gulp/**/*.js'
+    'scripts/gulp/**/*.js'
   ] );
-} );
+}
 
 /**
  * Lints the test js files for errors.
+ * @returns {Object} An output stream from gulp.
  */
-gulp.task( 'lint:tests', () => {
-  _genericLintJs( [
+function lintTests() {
+  return _genericLintJs( [
     'test/accessibility/*.js',
     'test/*.js'
```

Comment:
This could be simplified as `test/**/*.js`. Currently it's not catching any of the unit tests for atomic component

Reply:
Done! Good catch